packages/mds-provider/event-processor/index.ts

```diff
@@ -16,69 +16,118 @@
 
 import db from 'mds-db'
 import logger from 'mds-logger'
-import stream, { ReadStreamResult, ReadStreamOptions, StreamItem } from 'mds-stream'
+import stream, { ReadStreamOptions, StreamItem } from 'mds-stream'
 import uuid from 'uuid'
 import { isUUID } from 'mds-utils'
+import { VehicleEvent, VehicleEventPrimaryKey } from 'mds'
+import { StatusChange } from 'mds-db/dist/types'
 import { DeviceLabeler } from './labelers/device-labeler'
 import { ProviderLabeler } from './labelers/provider-labeler'
 import { StreamEntry } from './types'
-import { StatusChangesProcessor, StatusChangesProcessorStreamEntry } from './processors/status-changes-processor'
+import { StatusChangesProcessor } from './processors/status-changes-processor'
 import { TripLabeler } from './labelers/trip-labeler'
-import { TripsProcessor, TripsProcessorStreamEntry, TripEvent } from './processors/trips-processor'
+import { TripsProcessor, TripEvent } from './processors/trips-processor'
 
-const isStatusChangesProcessorStreamEntry = (entry: StreamEntry): entry is StatusChangesProcessorStreamEntry =>
+const isStatusChangesProcessorStreamEntry = (entry: StreamEntry): entry is StreamEntry<VehicleEvent> =>
```

Comment:
These functions should be broken out into a `validators.ts` file.

Reply:
They are only used in this one module. They are useful outside of the event processor context.